### PR TITLE
Better error handling for file not find exception during export

### LINKED ISSUE #1687: [Export] Don't treat FileNotFoundException as error

### PR DESCRIPTION
**Pull request by itaiad200**
closes #1687

### CODE REVIEW

File: clients/spark/core/src/main/scala/io/treeverse/clients/ExportStatus.scala [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package io.treeverse.clients
+
+// ExportStatus is the export status of a single file
+case class ExportStatus(path: String, success: Boolean, msg: String) extends Serializable {}
```
Comment (arielshaqed) [inline clients/spark/core/src/main/scala/io/treeverse/clients/ExportStatus.scala:4]: Ewww... That's a Go comment... 
Consider /** ... */. And it needs the imperative mode, without repeating the function name. 
🤷

File: clients/spark/core/src/main/scala/io/treeverse/clients/Exporter.scala [hunk 2]
```diff
@@ -133,31 +134,32 @@ object Exporter {
...
+            srcPath.getFileSystem(conf),
+            srcPath,
+            dstFS,
+            dstPath,
+            false,
+            conf)
+          null
+        } catch  {
+          case e : (FileNotFoundException) =>  ExportStatus(dstPath.toString, success = true, s"Unable to copy file ${dstPath.toString} from source ${srcPath.toString} since source file is missing: ${e.toString}")
+          case e : (IOException) =>  ExportStatus(dstPath.toString, success = false, s"Unable to copy file ${dstPath.toString} from source ${srcPath.toString}: ${e.toString}")
```
Comment (arielshaqed) [inline clients/spark/core/src/main/scala/io/treeverse/clients/Exporter.scala:159]: Do you really need all those toString calls when interpolating strings? (If so, possibly a reason to use % formatting)

**Review by arielshaqed — APPROVED** [review]
Cool!

File: clients/spark/core/src/main/scala/io/treeverse/clients/Exporter.scala (hunk 2 above)
Comment (itaiad200) [reply clients/spark/core/src/main/scala/io/treeverse/clients/Exporter.scala]: Done.

**Pull request merged**